src/ProductConstructionService/ProductConstructionService.Api/Api/v2018_07_16/Controllers/AssetsController.cs

```diff
@@ -96,8 +95,7 @@ public IActionResult GetDarcVersion()
         {
             version = version.Substring(0, lastPlus);
         }
-        return Ok(version);*/
-        return Ok("1.1.0-beta.24613.2");
+        return Ok(version);
```

Comment:
before doing this, we need to make sure that we're also publishing darc with the same version as, which we're not currently doing in the PCS pipeline. Merging pipelines is one way to do this I think

Reply:
Yeah, I would not roll out before we merge the pipelines. But I'd do this change separately

Reply:
The versions will match though because we will have the same production build number and revision